WordPress-Extra/ruleset.xml

```diff
@@ -2,6 +2,22 @@
 <ruleset name="WordPress Extra">
 	<description>Best practices beyond core WordPress Coding Standards</description>
 
+	<rule ref="Generic.PHP.DeprecatedFunctions"/>
+	<rule ref="Generic.PHP.ForbiddenFunctions"/>
+	<rule ref="Generic.Functions.CallTimePassByReference"/>
+	<rule ref="Generic.Formatting.DisallowMultipleStatements"/>
+	<rule ref="Generic.Formatting.MultipleStatementAlignment"/>
```

Comment:
We should probably have in the `README` a list of other sniffs that users may like to opt-in to, such as this, but which are perhaps even more opinionated than Extra already is.